profiler/Directory.Build.targets

```diff
@@ -12,4 +12,40 @@
       <AdditionalDependencies>$(LibDatadogDependencies);%(AdditionalDependencies)</AdditionalDependencies>
     </Link>
   </ItemDefinitionGroup>
+
+  <PropertyGroup>
+    <TopLevelDeployDirectoryBase Condition="'$(DeployToMonitoringHome)' == 'True'">$(EnlistmentRoot)\shared\bin\monitoring-home</TopLevelDeployDirectoryBase>
+    <TopLevelDeployDirectoryBase Condition="'$(DeployToMonitoringHome)' != 'True'">$(BuildOutputRoot)\DDProf-Deploy</TopLevelDeployDirectoryBase>
```

Comment:
I know this is just things moving around, but I'd _Really_ like to ditch a bunch of this soon if possible, and deploy everything to the "standard" directories instead (which are controlled in CI by env vars etc too). Just an aside, not suggesting we do anything about it now! 😄

Reply:
🙏 I'm always stressed out when I have to modify those files...